src/ContainerTerminal.jsx

```diff
@@ -172,10 +172,11 @@ class ContainerTerminal extends React.Component {
 
         // Show the terminal. Once it was shown, do not show it again but reuse the previous one
         if (!this.state.opened) {
-            this.state.term.open(this.terminalRef.current);
+            this.term.open(this.terminalRef.current);
+            this.term.loadAddon(new CanvasAddon());
             this.setState({ opened: true });
 
-            this.state.term.onData((data) => {
+            this.term.onData((data) => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1358-20230718-052024-6af1ca09-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)